@here/harp-mapview/lib/text/TextStyleCache.ts

```diff
@@ -334,25 +336,37 @@ export class TextStyleCache {
 
         const defaultLayoutParams = this.m_defaultStyle.layoutParams;
 
+        // Available only for PoiTechnique and LineMarkerTechnique, for TextTechnique
+        const hPlacement = getPropertyValue((technique as any).hPlacement, discreteZoomEnv) as
+            | string
+            | undefined;
+        const vPlacement = getPropertyValue((technique as any).vPlacement, discreteZoomEnv) as
+            | string
+            | undefined;
```

Comment:
Old code was wrong. Current `getPropertyValue` never returns `undefined`. If it's input was `undefined` it will return `null`. :/
Sorry, #1241 which corrected this typing got rejected.